pkg/dashboard/alert/alertStateManager.go

```diff
@@ -28,13 +30,17 @@ func NewAlertStateManager(
 	store store.Store,
 	alertEvaluationFrequencySeconds int,
 	thresholds map[string]threshold,
+	notificationsToken string,
+	host string,
 ) *AlertStateManager {
 	return &AlertStateManager{
-		notifManager: notifManager,
-		clientHub:    clientHub,
-		store:        store,
-		waitTime:     time.Duration(alertEvaluationFrequencySeconds) * time.Second,
-		thresholds:   thresholds,
+		notifManager:       notifManager,
+		clientHub:          clientHub,
+		store:              store,
+		waitTime:           time.Duration(alertEvaluationFrequencySeconds) * time.Second,
+		thresholds:         thresholds,
+		notificationsToken: notificationsToken,
```

Comment:
AlertStateManager handles the alert state
NotifManager handles the notifications.

AlertStateManager should not care about slack tokens.